src/navigation/components/NavItem.jsx

```diff
@@ -99,15 +99,14 @@ export const NavItem = props => {
 	);
 
 	const trigger = (
-		<Button
```

Comment:
The button was removed because the parent element already has the role of a button. If we were to use the button element, it would result in a nested role, which is not desirable.

Reply:
Changing button to div may cause other accessibility issues since div by default is not clickable element and should have `role="button"` attribute to act like button

Reply:
let me test it, maybe you right 🤔

Reply:
yeah its works fine 👍